pig-tinder-frontend/src/Components/Footer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Footer = () => {
+  return (
+    <div></div>
+  )
+}
+
+export default Footer
```

Comment:
Footer needs styles

Reply:
Yes, as i said in comments we pushed to fix resumes and sent to paul. Will fix styling and functionality at a later time. Just needed to push what has been completed.